code/__defines/subsystems.dm

```diff
@@ -30,6 +30,7 @@
 #define SS_INIT_MISC             11
 #define SS_INIT_SKYBOX           10
 #define SS_INIT_MAPPING          9
+#define SS_INIT_DEPARTMENTS		 9
```

Comment:
Will need to increment the other priorities above this one, or just use an existing define.

Reply:
Oh, I think there was another pair of defines that shared a priority so I thought it would be okay. I better look at that one as well.